pennylane/devices/qubit/measure.py

```diff
@@ -19,13 +19,15 @@
 from scipy.sparse import csr_matrix
 
 from pennylane import math
+from pennylane import numpy as np
```

Comment:
I'm also now wondering why we are using `pennylane.numpy` instead of vanilla numpy.  Seems like in this case we might actually just want vanilla numpy.

Reply:
I'm not used to distinguishing between the two. Since at the moment this is used when using `parameter-shift`, I guess vanilla numpy would do. In the future, I would be curious to know whether you think this can be auto-differentiated given that we do not need SciPy's sparse routines anymore.